.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+"on":
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  Test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+          - "3.6"
```

Comment:
Should the minimum version be 3.7 as the 3.6 is end of its life cycle?

Reply:
Unfortunately not – we still need to support 3.6 for certain environments.